### PR TITLE
Revert "correcting the wrong Class name to right Class Name along with creating the convert_tags_to_color function

### PR DESCRIPTION
**Pull request by achiverram28**

### CODE REVIEW

**Comment by achiverram28** [comment]
Can you just check now?

File: README.md [hunk 1]
```diff
@@ -43,9 +43,9 @@ printer.configure_options(PrinterConfig(delimiter_left="|", delimiter_right="|",
 The `Printer` class also has a `print()` method that can be used to print text. The `print()` method can be used as follows:
 
 ```python
-from fancyprint import Printer
-from fancyprint.helpers.types import PrintConfig
-from fancyprint.helpers.constants import Align
+from src.fancyprint import Printer
+from src.fancyprint.helpers.types import PrintConfig
+from src.fancyprint.helpers.constants import Align
 printer=Printer()
```
Comment (IamMU) [inline README.md]: It's still a package, remove the src. We don't need it.

**Review by IamMU — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (achiverram28) [reply README.md]: ok

**Comment by achiverram28** [comment]
now?

**Comment by IamMU** [comment]
@achiverram28 Epic work :fire:

**Comment by achiverram28** [comment]
Thank you for giving this opportunity

**Comment by achiverram28** [comment]
If you have any new issues , I can work on it

**Comment by achiverram28** [comment]
967CDB95-CC29-4E49-8E87-C0E289228CC0
Why is it showing Printer when I corrected it to Print?

**Comment by IamMU** [comment]
> If you have any new issues , I can work on it

You'll have to see the issues tab, currently no issues that are free. But you are free to make more issues and work on them. 
New feature ideas are appreciated :slightly_smiling_face:

**Comment by IamMU** [comment]
> 967CDB95-CC29-4E49-8E87-C0E289228CC0
> Why is it showing Printer when I corrected it to Print?

oof I think I messed that one up in resolving, not a big deal, I'll fix it in a moment.

**Pull request merged**